src/app/spreadsheet/ExcelFileUtilities.ts

```diff
@@ -0,0 +1,81 @@
+import { saveAs } from "file-saver";
```

Comment:
this file already exists in utilities so re-use it instead of creating duplicate